scripts/sourceversion.py

```diff
@@ -1,9 +1,10 @@
 import subprocess
 import sys
+import pytz
```

Comment:
pytz is external lib, either add it somewhere as dep or chatgpt:

``` import os
from datetime import datetime, timezone, timedelta

def get_time_in_timezone(timezone_offset):
    return datetime.now(timezone.utc) + timedelta(hours=timezone_offset)

if __name__ == '__main__':
    # Bratislava is in the Central European Time (CET) zone, which is UTC+1, and Central European Summer Time (CEST), which is UTC+2.
    # You can adjust this based on the current date.
    # For simplicity, we'll use UTC+1 here. You might want to add logic to handle daylight saving time.
    cet_offset = 1
    ts = get_time_in_timezone(cet_offset)
    print(f"This info was generated on: {ts.strftime('%Y-%m-%d %H:%M:%S %Z%z')}")
```

Reply:
the solution itself says it is not very good, because it does not handle daylight saving time. I will add pytz as dep, since you suggested that option too.

Reply:
after discussion, using proposed solution